### PR TITLE
fix: pinInput demo box width

### PR DESCRIPTION
**Pull request by harshndroid**
## MOLECULE/PININPUT
**TASK**: 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### CODE REVIEW

File: components/atom/pinInput/demo/ArticleChildren.js [hunk 1]
```diff
@@ -91,55 +91,55 @@ const ArticleChildren = ({className}) => {
           <PinInput onChangeHandler={onChangeHandler} defaultValue={cardCode}>
             <Grid cols={19} gutter={[8, 8]}>
               <Cell>
-                <PinInputField isFullWidth />
+                <PinInputField isFullWidth={false} />
```
Comment (turolopezsanabria) [inline components/atom/pinInput/demo/ArticleChildren.js]:
```suggestion
                <PinInputField/>
```

Comment (turolopezsanabria) [reply components/atom/pinInput/demo/ArticleChildren.js]: I suggest not including the optional `Prop` at all

Comment (harshndroid) [reply components/atom/pinInput/demo/ArticleChildren.js]: @turolopezsanabria got it! will try it out

Comment (harshndroid) [reply components/atom/pinInput/demo/ArticleChildren.js]: @turolopezsanabria without including any prop i'll have to make another component for this case, right?

Comment (harshndroid) [reply components/atom/pinInput/demo/ArticleChildren.js]: @turolopezsanabria i've committed the changes without making new component.

**Pull request closed without merging**